Marlin/src/lcd/menu/menu_tramming.cpp

```diff
@@ -42,20 +42,18 @@
 float z_measured[G35_PROBE_COUNT] = { 0 };
 static uint8_t tram_index = 0;
 
-bool probe_single_point() {
+static bool probe_single_point() {
   do_blocking_move_to_z(TERN(BLTOUCH, Z_CLEARANCE_DEPLOY_PROBE, Z_CLEARANCE_BETWEEN_PROBES));
   // Stow after each point with BLTouch "HIGH SPEED" mode for push-pin safety
   const float z_probed_height = probe.probe_at_point(screws_tilt_adjust_pos[tram_index], TERN(BLTOUCH_HS_MODE, PROBE_PT_STOW, PROBE_PT_RAISE), 0, true);
   DEBUG_ECHOLNPAIR("probe_single_point: ", z_probed_height, "mm");
   z_measured[tram_index] = z_probed_height;
-
-  // Move XYZ to wait position
```

Comment:
Shouldn't we keep the comments?

Reply:
The function name is descriptive enough that the comment is redundant.